dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionService.java

```diff
@@ -54,13 +54,24 @@ public Integer processContributionUpdate(UpdateLogContributionRequest updateLogC
     @Timed(value = "laa_dces_drc_service_process_contributions_daily_files",
             description = "Time taken to process the daily contributions files from DRC and passing this for downstream processing.")
     public boolean processDailyFiles() {
-        Map<String, CONTRIBUTIONS> successfulContributions = new HashMap<>();
-        Map<String, String> failedContributions = new HashMap<>();
-        // get all the values to process via maat call
-        List<ConcurContribEntry> contributionsList = contributionClient.getContributions("ACTIVE");
-        sendContributionsToDrc(contributionsList, successfulContributions, failedContributions);
+        int defaultNoOfRecord = feature.noOfContributionRecords();
+        List<ConcurContribEntry> contributionsList;
+        int startingId = 0;
+        boolean isSuccessful = false;
```

Comment:
This seems to be true if any records got processed (i.e. it's only false if no records were sent to the DRC and no contributions were updated/no contribution-file created).

So maybe a better name would be `hasProcessedAnyRecords` than `isSuccessful`?

Really minor, but this method can successfully do nothing if there's no ACTIVE records left to process (in which case it will return `false`).